sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/CommonTestFramework/EventSourceTestHelper.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests.CommonTestFramework;
+
+internal static class EventSourceTestHelper
```

Comment:
add a comment linking to Otel, if this inspired by OTel tests.!

Reply:
I reached out to CELA last week and they didn't think a citation was necessary. I'll double check just to confirm we're doing the right thing here.

It's worth noting though that OTel copied these classes from Application Insights (without citation).
Who would you recommend we cite here - OTel or AI?

Reply:
i wasn't commenting based on the legal aspects! Purely for the purpose of fixing the original one if something new was discovered here!

Reply:
added citations.